quickwit-index/src/actors/packager.rs

```diff
@@ -223,8 +250,9 @@ impl SyncActor for Packager {
     ) -> Result<(), quickwit_actors::ActorExitStatus> {
         commit_split(&mut split, ctx)?;
         let segment_metas = merge_segments_if_required(&mut split, ctx)?;
+        let tags = extract_tags(&mut split)?;
```

Comment:
I think we can move that call into create_packaged_split and remove one argument.